app/resources/v1/pacbio/library_batch_resource.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module V1
+  module Pacbio
+    # This resource is used to create batches of libraries in one go.
+    #
+    # @example
+    #  {
+    #     "data": {
+    #       "type": "library_batches",
+    #       "attributes": {
+    #         "libraries_attributes": [
+    #           {
+    #             "volume": 50.2,
+    #             "concentration": 2.222,
+    #             "template_prep_kit_box_barcode": "LK1234567",
+    #             "insert_size": 100,
+    #             "pacbio_request_id": 1,
+    #             "tag_id": 1,
+    #             "primary_aliquot_attributes": {
+    #               "volume": 50.2,
+    #               "concentration": 2.222,
+    #               "template_prep_kit_box_barcode": "LK1234567",
+    #               "insert_size": 100,
+    #               "tag_id": 1
+    #             },
+    #           }
+    #         ]
+    #       }
+    #   }
+    #
+    # @note Access this resource via the `/v1/pacbio/library_batches/` endpoint.
+    #
+    # Provides a JSON:API representation of {Pacbio::LibraryBatch}.
+    #
+    # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
+    # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package
+    # for the service implementation of the JSON:API standard.
+    class LibraryBatchResource < JSONAPI::Resource
```

Comment:
I think we may need to add some relationships here? We will need to return the tube barcodes and also add that into the documentation.

Reply:
There is a test case for this. When doing a post you can include relationships, if we do post with `?include='libraries.tube'` we will get the tubes back.

Reply:
Ok. That's great but could we document it in the resource? 

I am talking to Kieron Taylor at the moment and he would like to start using our endpoints but the docs are sketchy or missing so having something explicit in there would help a lot. I think we will also start adding it to other resources.

e.g. I am looking at pacbio pooling and I have found how to retrieve the includes and filter but did that by looking at the ui, resource and network in developer tools. I think having these things at the top of the resource will make it much easier.

As part of my story I plan to add it for this example.

Reply:
Ah I see. Ok sure ill add a comment on it.